targets/UnrealMarketplacePlugin/make.js

```diff
@@ -14,35 +13,54 @@ var copyright =
-    var sdkVersion = exports.sdkVersion;
-    var buildIdentifier = exports.buildIdentifier;
-
-    // Create the Source folder in the plugin with all the modules
-    bpMakeJsPath.makeBpCombinedAPI(apis, copyright, sourceDir, apiOutputDir, ueTargetVersion, sdkVersion, buildIdentifier);
-    cppMakeJsPath.makeCppCombinedAPI(apis, copyright, sourceDir, apiOutputDir, ueTargetVersion, sdkVersion, buildIdentifier);
-    commonMakeJsPath.makeCommonCombinedAPI(apis, copyright, sourceDir, apiOutputDir, ueTargetVersion, sdkVersion, buildIdentifier);
-
-    var authMechanisms = getAuthMechanisms(apis);
-    var locals = {
-        apis: apis,
-        ueTargetVersion: ueTargetVersion,
-        sdkVersion: sdkVersion,
-        copyright: copyright,
-        hasClientOptions: authMechanisms.includes("SessionTicket"),
-        hasServerOptions: authMechanisms.includes("SecretKey")
-    };
-
-    // Copy the resources, content and the .uplugin file
-    templatizeTree(locals, path.resolve(sourceDir, "source/PlayFab/Content"), path.resolve(apiOutputDir, "PlayFabPlugin/PlayFab/Content"));
-    templatizeTree(locals, path.resolve(sourceDir, "source/PlayFab/Resources"), path.resolve(apiOutputDir, "PlayFabPlugin/PlayFab/Resources"));
-    templatizeTree(locals, path.resolve(sourceDir, "source/PlayFab/PlayFab.uplugin.ejs"), path.resolve(apiOutputDir, "PlayFabPlugin/PlayFab/PlayFab.uplugin.ejs"));
-
-    // Create the Example project folder
-    templatizeTree(locals, path.resolve(sourceDir, "examplesource"), apiOutputDir);
-
-    // Copy the PlayFabPlugin folder just created into the ExampleProject
-    templatizeTree(locals, path.resolve(apiOutputDir, "PlayFabPlugin"), path.resolve(apiOutputDir, "ExampleProject/Plugins"));
-}
\ No newline at end of file
+exports.makeCombinedAPI = function (apis, sourceDir, baseApiOutputDir) {
+    // The list of current supported UE versions - Intended to be the latest 3
+    var ueTargetVersions = ["4.18", "4.19", "4.20"];
+
+    for (var v = 0; v < ueTargetVersions.length; v++) {
+        var ueTargetVersion = ueTargetVersions[v];
+        var apiOutputDir = path.resolve(baseApiOutputDir, ueTargetVersion); // Break multiple versions into separate top level folders
+
+        console.log("Generating Unreal Plugin to " + apiOutputDir);
+
+        // Create the Source folder in the plugin with all the modules
+        bpMakeJsPath.makeBpCombinedAPI(apis, copyright, sourceDir, apiOutputDir, ueTargetVersion, exports.sdkVersion, exports.buildIdentifier);
+        cppMakeJsPath.makeCppCombinedAPI(apis, copyright, sourceDir, apiOutputDir, ueTargetVersion, exports.sdkVersion, exports.buildIdentifier);
+
+        var authMechanisms = getAuthMechanisms(apis);
+        var locals = {
+            apis: apis,
+            buildIdentifier: exports.buildIdentifier,
+            copyright: copyright,
+            enumTypes: collectEnumsFromApis(apis),
+            errorList: apis[0].errorList,
+            errors: apis[0].errors,
```

Comment:
Any particular reason why you want to use index 0 of apis?

Reply:
Not specifically.
Every api document is expected to have a full copy of that particular information. It's copy paste in every API.
For now, you could pick list.any(), but that isn't a thing in JS.